configure-system-libraries.sh

```diff
@@ -37,11 +36,18 @@ patch_config()
 		if [ -L "bin/${REPLACE}" ]; then
 			return 0
 		fi
-
+		KERNEL="$(uname -s)"
+		ARCH="$(arch)"
 		printf "Searching for %s... " "${LABEL}"
 		for DIR in ${SEARCHDIRS} ; do
 			for LIB in ${SEARCH}; do
 				if [ -f "${DIR}/${LIB}" ]; then
+					# x86-64 Linux might have 32-bit libraries present, link against the 64 bit libraries since there is no 32-bit Linux release of dotnet
```

Comment:
This function is called by both Linux and macOS, so implementing Linux-specific checks here isn't great. Can we please handle this insteacd at the outer level, the same way as the macOS case?